R/build-articles.R

```diff
@@ -144,19 +139,12 @@
 #'   extension: pdf
 #' ```
 #'
-#' If you want to set an output format for all your articles, you can do that
-#' by adding a `vignettes/_site.yml`, much like you would for an
-#' [rmarkdown website](https://rmarkdown.rstudio.com/docs/reference/render_site.html).
-#' For example, you can backport some bookdown features such as cross-references
-#'  to all your articles by using the
-#' [bookdown::html_document2](https://bookdown.org/yihui/bookdown/a-single-document.html)
-#' format.
-#'
-#' ```yaml
-#' output:
-#'   bookdown::html_document2:
-#'   number_sections: false
-#' ```
+#' To work with pkgdown, the output format must accept `template`, `theme`, and
```

Comment:
Add example of `html_vignette()` as output format that doesn't?

Reply:
It's supported by special casing it.

Reply:
But there's a test with "# theme is not set since html_vignette doesn't support it", what am I missing?

Reply:
See line 256 below